src/main/java/seedu/address/logic/commands/AddCommand.java

```diff
@@ -23,27 +23,27 @@
     public static final String COMMAND_WORD = "add";
     public static final String COMMAND_ALIAS = "a";
 
-    public static final String MESSAGE_USAGE = COMMAND_WORD + ": Adds a tutor or student to the database. "
+    public static final String MESSAGE_USAGE = COMMAND_WORD + ": Adds a tutor or student to the database. \n"
             + "Parameters: "
-            + "<s "
+            + "<t "
             + PREFIX_NAME + "NAME "
             + PREFIX_PHONE + "PHONE "
             + PREFIX_GENDER + "GENDER "
+            + PREFIX_QUALIFICATION + "QUALIFICATION "
             + "[" + PREFIX_REMARK + "REMARK] "
-            + PREFIX_TAG + "TAG>"
-            + " or <t "
+            + PREFIX_TAG + "TAG...>\n"
+            + "or <s "
             + PREFIX_NAME + "NAME "
             + PREFIX_PHONE + "PHONE "
             + PREFIX_GENDER + "GENDER "
-            + PREFIX_QUALIFICATION + "QUALIFICATION "
             + "[" + PREFIX_REMARK + "REMARK] "
-            + PREFIX_TAG + "TAG...>\n"
+            + PREFIX_TAG + "TAG>\n"
```

Comment:
Students can also have multiple tags:
```suggestion
            + PREFIX_TAG + "TAG...>\n"
```